cmd/cli/command/access/ensure.go

```diff
@@ -89,6 +103,45 @@ var ensureCommand = cli.Command{
 				ent.Duration = durationpb.New(duration)
 			}
 			req.Entitlements = append(req.Entitlements, ent)
+
+			gConf, err := grantedConfig.Load()
+			if err != nil {
+				return err
+			}
+
+			startURL := gConf.CommonFateDefaultSSOStartURL
+
+			region := gConf.CommonFateDefaultSSORegion
+
+			pruneStartURLs := []string{startURL}
+
+			g := awsconfigfile.Generator{
+				Config:              awsConfig,
+				ProfileNameTemplate: awsconfigfile.DefaultProfileNameTemplate,
+				NoCredentialProcess: false,
+				Prefix:              "",
+				PruneStartURLs:      pruneStartURLs,
+			}
+
+			ps := profilesource.Source{
+				Entitlements: req.Entitlements,
+				SSORegion:    region,
+				StartURL:     startURL,
+				DashboardURL: cfg.APIURL,
+				Client:       accountClient,
+			}
+			g.AddSource(ps)
+			clio.Info("Updating your AWS config file (~/.aws/config) with profiles from Common Fate...")
```

Comment:
Rather than dumping all profiles can you just make it add the one they requested? This should make conflicting profiles less likely to occur, and should improve the UX if there *is* a conflicting profile